spec/factories/seasons.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :season do
+    team { build(:team) }
+    # games { build(:game) } Is this a typo? Should each game belong to a season? Or can a game be a part of different seasons?
```

Comment:
A season has many games.  This current season has 8 games.  Is the model relationship set up incorrectly?